docker-compose.yml

```diff
@@ -20,16 +20,25 @@ services:
       dockerfile: Dockerfile
     container_name: appointment
     environment:
-      - MONGODB_URL=${MONGOURL}
       - PORT=3012
+      - MONGOADMIN=${MONGOADMIN}
+      - MONGOPASS=${MONGOPASS}
+      - MONGOURL=${MONGOURL}
+      - NODE_ENV=production
+      - RABBIT_USERNAME=${RABBIT_USERNAME}
+      - RABBIT_PASSWORD=${RABBIT_PASSWORD}
+      - RABBIT_HOST=${RABBIT_HOST}
+      - RABBIT_PORT=${RABBIT_PORT}
+      - RABBIT_VHOST=${RABBIT_VHOST}
+      - API_PREFIX=/api/v1
+      - JWT_SECRET=${JWT_SECRET}
+      - KAFKA_HOST=${KAFKA_HOST}
```

Comment:
need to add dragonfly service

Reply:
also add the readme from template in this repo

Reply:
Resolved in the commit `62348a81a7e974d7714c4058255e5ae8da78d204`